tools/manifest-templator/manifest-templator.go

```diff
@@ -82,7 +82,7 @@ var (
 func check(err error) {
 	if err != nil {
 		log.Println("error: ", err)
-		panic(err)
+		os.Exit(1)
```

Comment:
since we have here a defer function above the caller, the last comment is more significant